src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardPerustelut.js

```diff
@@ -418,9 +418,10 @@ const MuutospyyntoWizardPerustelut = ({
                     muutosperustelut={R.sortBy(R.prop("koodiArvo"))(
                       muutosperustelut.muutosperusteluList
                     )}
-                    stateObject={R.path(["perustelut", "opiskelijavuodet"])(
-                      muutoshakemus
-                    )}
+                    stateObject={{
+                      opiskelijavuodet: R.path(["opiskelijavuodet"], muutoshakemus),
+                      perustelut: R.path(["perustelut"],muutoshakemus)
```

Comment:
Do we need to pass on the whole perustelut object here? It contains reasoning data of all the form sections.

Reply:
I think not. I'll remove it.